### PR TITLE
Transition content sharing docs to v1 from beta

### PR DESCRIPTION
**Pull request by benlee-msft**

### CODE REVIEW

File: api-reference/v1.0/resources/calloptions.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+# callOptions resource type
+
+Namespace: microsoft.graph
+
+An abstract base class that contains the optional features for a call.
+
+## Properties
+
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/calloptions.md:17]: Should the property "hideBotAfterEscalation" also be listed under Properties?

image

**Comment by JarbasHorst** [comment]
Hello @benlee-msft, I reviewed your PR and left one comment. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [18577](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18577) into your branch to include my fixes and before addressing my comments.

File: api-reference/v1.0/resources/calloptions.md (hunk 1 above)
Comment (benlee-msft) [reply api-reference/v1.0/resources/calloptions.md:17]: Yes you are right I'll add that in

**Comment by benlee-msft** [comment]
@JarbasHorst Addressed comments and merged your edits. Thanks!

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [18601](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18601) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**